src/pages/home/report/ReportActionItem.js

```diff
@@ -328,6 +340,11 @@ function ReportActionItem(props) {
         return (
             <>
                 {children}
+                {!_.isEmpty(props.action.linkMetadata) && (
+                    <View style={props.draftMessage ? styles.chatItemReactionsDraftRight : {}}>
+                        <LinkPreviewer linkMetadata={_.filter(props.action.linkMetadata, (item) => !_.isEmpty(item))} />
```

Comment:
👋 This has caused a regression in https://github.com/Expensify/App/issues/21195
We shouldn't show link preview if message was flagged as inappropriate, this was resolved by hiding `LinkPreview` if `isHidden=true`